### ISSUE TITLE
4.7.3 crashes directly at start if terrain maps are enabled

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/OsmAnd#19263]
### Description

The new beta version OsmAnd 4.7.3 crashes directly on startup. The map is shown for maybe a second before the crash, no real interaction is possible.

The previous version 4.7.1 didn't crash.

### Steps to reproduce

Start OsmAnd. 

### Actual result

Crash. 

### Expected result

No Crash. 

### Your Environment (required)

```
OsmAnd Version: 4.7.3
Android/iOS version: Android 10
Device model: Samsung S9
Crash-Logs: no crash logs present, osmandcore.log is empty
```

**Comment by pebogufi** [comment]
@scaidermern  
But you could install it ???
See https://github.com/osmandapp/OsmAnd/issues/19262
Exactly, which date/time and subversion #
was this install ?

**Comment by scaidermern** [comment]
> @scaidermern But you could install it ??? See #19262 Exactly, which date/time and subversion # was this install ?

Yes, I guess because this _isn't_ the nightly. It is the official playstore _beta_ version. I installed it yesterday evening.

**Comment by scaidermern** [comment]
> See my comment here [#17805 (comment)](https://github.com/osmandapp/OsmAnd/issues/17805#issuecomment-1981633706). I think the way we compiled it was changed, **but also the signature**, and I think the latter causes the issue, and I am not convinced is actually necessary.

I guess this comment was meant for #19262 instead? A changed signature shouldn't lead to a crash.

**Comment by scaidermern** [comment]
I've found a way to resolve my crash. I've tried to move various files away from the OsmAnd storage directory to start with less features. The crash is caused by the heightmap feature. Removing/renaming the `geotiff` directory fixes the crash. As soon as I download the terrain map (3D) (for Germany, in my case), the crash happens again. Immediately when drawing the map.

**Comment by Chumva** [comment]
@scaidermern do you have any crash logs?

**Comment by MatthieuQ** [comment]
Hello, 
I don't know if it's related to the terrain maps, but the same observation was made today with beta version 4.7.3 on Android 14, Samsungs A52s.
The app crashes immediately.

**Comment by scaidermern** [comment]
> Hello, I don't know if it's related to the terrain maps, but the same observation was made today with beta version 4.7.3 on Android 14, Samsungs A52s. The app crashes immediately.

Try to remove (or rename) your `geotiff` directory that is located inside the OsmAnd storage directory.

**Comment by cmschneider1966** [comment]
I have Had exactly the same Problem since the Last Update this morning. And the same procedure works for me as for scaidermen. Rename or move geotiff and OsmAnd works. With crash there ist normal Error message. The App ist simply closed.

**Comment by scaidermern** [comment]
Any chance to have a fixed version soon? This crash affects many users.

**Comment by scaidermern** [comment]
> It was claimed to be fixed by [osmandapp/OsmAnd-core#692](https://github.com/osmandapp/OsmAnd-core/pull/692), so that does not fix it?

I guess/hope it does fix it. But so far there is no new beta release with this fix included. Leading to more and more users complaining about no longer being able to start OsmAnd.

**Comment by scaidermern** [comment]
4.7.4 is available now and the crash is gone 👍
